backend/src/test/java/backend/mulkkam/cup/service/CupServiceIntegrationTest.java

```diff
@@ -188,4 +193,149 @@ void success_withExistedMemberId() {
             });
         }
     }
+
+    @DisplayName("컵을 수정할 때에")
+    @Nested
+    class Modify {
+
+        @DisplayName("컵 이름 및 용량이 수정된다")
+        @Test
+        void success_withValidData() {
+            // given
+            Member member = new MemberFixture().build();
+            memberRepository.save(member);
+
+            String beforeCupNickName = "변경 전";
+            String afterCupNickName = "변경 후";
+            Integer beforeCupAmount = 500;
+            Integer afterCupAmount = 1000;
+
+            Cup cup = new CupFixture()
+                    .member(member)
+                    .cupAmount(new CupAmount(beforeCupAmount))
+                    .cupNickname(new CupNickname(beforeCupNickName))
+                    .build();
+
+            Cup savedCup = cupRepository.save(cup);
+            CupNicknameAndAmountModifyRequest cupNicknameAndAmountModifyRequest = new CupNicknameAndAmountModifyRequest(
+                    afterCupNickName,
+                    afterCupAmount
+            );
+
+            // when
+            cupService.modifyNicknameAndAmount(
+                    savedCup.getId(),
+                    member.getId(),
+                    cupNicknameAndAmountModifyRequest
+            );
+
+            Cup changedCup = cupRepository.findById(savedCup.getId())
+                    .orElseThrow();
+
+            // then
+            assertSoftly(softly -> {
+                softly.assertThat(changedCup.getCupAmount().value()).isEqualTo(afterCupAmount);
+                softly.assertThat(changedCup.getNickname().value()).isEqualTo(afterCupNickName);
+            });
+        }
+
+        @DisplayName("수정할 컵만 변경된다")
+        @Test
+        void success_onlyOneCupShouldBeModified() {
```

Comment:
```suggestion
        void success_whenOneCupIsModified() {
```

사소한 부분이지만, 이런 식으로 조건을 명시하는 것은 어떻게 생각하시나요~?
현재 메서드명은 결과에 초점이 맞춰져 있어서, 조건을 드러내면 더 명확할 것 같아여

Reply:
해당 부분에서 테스트하고 싶었던 것은

멤버가 2명일 때 바꾸고 싶은 멤버의 컵만 바뀌는 것을 검증하고자 했습니다.

현재 제 테스트 메서드명도 딱히 좋진 않지만 ㅋㅋㅋ

칼리가 제시해주신 조건은 하나의 컵만 바뀐다는 것은 뭔가 명확하지 않아서

`success_whenCertainMembersCupChanges` 이렇게 바꾸어보았는데, 어떤가요 ?

Reply:
`success_whenCertainCupChanges`으로 변경 완